src/common/common.cpp

```diff
@@ -102,7 +102,7 @@ static bool no_visible_owner(HWND window) noexcept
     return rect.top == rect.bottom || rect.left == rect.right;
 }
 
-FancyZonesFilter get_fancyzones_filtered_window(HWND window)
+FancyZonesFilter get_fancyzones_filtered_window(HWND window, bool IsNewlyCreatedWindow)
```

Comment:
I'm not convinced is a best thing to pass `IsNewlyCreatedWindow` to `get_fancyzones_filtered_window` since `IsNewlyCreatedWindow` is already known by the caller, the caller can simply skip calling `get_fancyzones_filtered_window` if it knows it won't zone the window in the first place.

Reply:
Idea was to set `.zonable` attribute accordingly and then it could be used in the future e.g. like `get_fancyzones_filtered_window` is used to check for `.no_visible_parent` to decide whether to preserve window history or not.

Reply:
The purpose of `get_fancyzones_filtered_window` is to process the input window and extract information that the caller will use to decide how to proceed, while `IsNewlyCreatedWindow` is an already known thing by the caller. The caller should not call `get_fancyzones_filtered_window` if the result of the call is determined by the value of `IsNewlyCreatedWindow`.